src/api/timetable/entity.ts

```diff
@@ -35,3 +43,7 @@ export interface TimetableAddLectureRequest {
 export interface TimetableRemoveLectureResponse extends APIResponse {
   [index: number]: TimetableLectureInfo;
 }
+
+export interface LastUpdatedDateResponse extends APIResponse {
+  [index: number]: TimetableVersionInfo;
+}
```

Comment:
그 역할은 아마 여기서 해줄 수 있을 것 같네요

현재 이 Response는 배열 구조를 반환하는거예요.
api의 response가 
```ts
response = [
  TimetableVersionInfo객체,
  TimetableVersionInfo객체
]
```
처럼 올 때 쓰는거지여

아마 이걸 원하는 방식으로 바꾸려면, 요걸 수정하면 data의 타입이 잡혀줄거예요~
```suggestion
export interface LastUpdatedDateResponse extends APIResponse {
  id: string;
  version: string;
  type: string;
  created_at: string;
  updated_at: string;
}
```

Reply:
이렇게 바꾸고나면 `data: updatedDate` 의 타입을 확인했을때 원하는 타입과 가까워졌을거예요

Reply:
확실히 이렇게 하니까 updatedDate 타입에 어떤 게 있는지 쉽게 확인이 가능하네요!! 반영했습니다!
c3c0e69840e42a5f7cc82c9afe787100f6874332